charts/pulsar/templates/presto/presto-worker-statefulset.yaml

```diff
@@ -130,6 +130,11 @@ spec:
           {{- end }}
           command: ["sh", "-c"]
           args:
+            {{ if .Values.presto.worker.commandOverride }}
+            {{ with .Values.presto.worker.commandOverride }}
+            -{{- toYaml . | indent 12 }}
+            {{- end }}
+            {{ else }}
```

Comment:
The same whitespace control question as above.

Reply:
done